packages/edit-site/src/components/sidebar-navigation-screen-navigation-item/index.js

```diff
@@ -49,7 +49,14 @@ export default function SidebarNavigationScreenNavigationItem() {
 				/>
 			}
 			content={
-				post ? decodeEntities( post?.description?.rendered ) : null
+				<>
+					<a href={ post?.link } target="_blank" rel="noreferrer">
```

Comment:
Is there a component we can use instead of a normal link?

We need to change the style of this.

Reply:
a11y-wise it's best not to open links in a new tab without warning:

> It is recommended that when links are opened to a new window, there is advance warning.
> https://www.w3.org/TR/WCAG20-TECHS/G200.html

Using the `<Button />` element doesn't look like it automatically handles adding a visually-hidden warning if you pass an href and a `target="_blank"`, but there may be another component that does. I'm looking.

Reply:
Ah, right. The the component @richtabor mentioned in the issue: `ExternalLink` lol. I'll update the PR.